mysql/README.md

```diff
@@ -0,0 +1,233 @@
+## MySQL Example Usage
```

Comment:
I think we should have a top-level header and a bit of explanatory text. How about something like:

# HarbourBridge: Turnkey MySQL-to-Spanner Evaluation

HarbourBridge is a stand-alone open source tool for Cloud Spanner evaluation, using data from an existing PostgreSQL or MySQL database. This README provides details of the tools MySQL capabilities. For
general HarbourBridge information see <pointer-to-main-README>.

## Example MySQL Usage

Reply:
Done.